db/seeds.rb

```diff
@@ -86,4 +87,79 @@
       confirmed: "no"
     }
   )
+
+  # Sole trader
```

Comment:
Reckon this is fine for this PR, but we need to think about a different approach going forward especially as @timstone is now busy putting together his list of registrations he'd like to see seeded!

Reply:
Agreed, this could get messy pretty quickly – should we have a story to look into how to handle seeding in a sustainable way?

Reply:
Agreed and done! Implement sustainable method for seeding the database